analysed-packages/ca-certificates/version-20211016/ca-certificates-20211016-SPDX2TV.spdx

```diff
@@ -0,0 +1,2046 @@
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseComments: <text> The information in the file is:
+This file is distributed under the same license as the ca-certificates package.
+The copyright file states:
+Files: debian/*
+examples/*
+Makefile
+mozilla/*
+sbin/*
+Copyright: 2003 Fumitoshi UKAI 
+2009 Philipp Kern 
+2011 Michael Shuler 
+Various Debian Contributors
+License: GPL-2+
+Thus we concluded GPL-2.0-or-later as license of this file </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Same-license-as
+LicenseInfoInFile: LicenseRef-free-unknown
+FileCopyrightText: <text> Copyright (C) 2005-2007 Software in the Public Interest
+Copyright (C) 2005-2007 Software in the Public Interest </text>
+
+##File
+
+FileName: ca-certificates-debian-20211016.tar.gz/ca-certificates-debian-20211016.tar/ca-certificates-debian-20211016/debian/po/nl.po
+SPDXID: SPDXRef-item700605
+FileChecksum: SHA1: ea322376159cb2d26bd2a3d14ffe0a938accc217
+FileChecksum: SHA256: ce6249b05e11bee703113071f8f2903d331981108791b5000eaac95cc3b23954
+FileChecksum: MD5: a4f4d5adba87c53d81cd56726bbed8f1
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseComments: <text> The information in the file is:
+This file is distributed under the same license as the ca-certificates package.
+The copyright file states:
+Files: debian/*
+examples/*
+Makefile
+mozilla/*
+sbin/*
+Copyright: 2003 Fumitoshi UKAI 
+2009 Philipp Kern 
+2011 Michael Shuler 
+Various Debian Contributors
+License: GPL-2+
+Thus we concluded GPL-2.0-or-later as license of this file </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Same-license-as
+LicenseInfoInFile: LicenseRef-free-unknown
+FileCopyrightText: <text> Copyright (C) 2011 Frans Pop <aragorn@tiscali.nl>, 2004, 2006. Frans Pop <elendil@planet.nl>, 2007. Jeroen Schot <schot@a-eskwadraat.nl>, 2011. </text>
+
+##File
+
+FileName: ca-certificates-debian-20211016.tar.gz/ca-certificates-debian-20211016.tar/ca-certificates-debian-20211016/debian/po/ru.po
+SPDXID: SPDXRef-item700597
+FileChecksum: SHA1: 6bfe998d367aeaa32e74f6d7bb27111f86925f92
+FileChecksum: SHA256: 4ee48724d3bf2b40b61338c4d8fdc825ab259fc0c0bc10328f84f59a96aae535
+FileChecksum: MD5: 4f5067144d222f484aba2d5cad218b7a
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseComments: <text> The information in the file is:
+This file is distributed under the same license as the ca-certificates package.
+The copyright file states:
+Files: debian/*
+examples/*
+Makefile
+mozilla/*
+sbin/*
+Copyright: 2003 Fumitoshi UKAI 
+2009 Philipp Kern 
+2011 Michael Shuler 
+Various Debian Contributors
+License: GPL-2+
+Thus we concluded GPL-2.0-or-later as license of this file </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Same-license-as
+LicenseInfoInFile: LicenseRef-free-unknown
+FileCopyrightText: <text> Copyright (C) 2003 Ilgiz Kalmetev <translator@ilgiz.pp.ru>
```

Comment:
@ParianGol  in this file I see only the following copyright statements:
Copyright (C) 2003 Ilgiz Kalmetev <translator@ilgiz.pp.ru>
Copyright (C) 2007 Mikhail Gusarov <dottedmag@dottedmag.net> Yuri Kozlov <yuray@komyakino.ru>, 2011.
In your SDPX Tag Value file these are dublicated. 
Can it be that these were imported as "user Findings" during the action import report? Please delete all the "User Findings" and regenerate the documents
Please keep in mind that there are more file affected than I looked into